slim/export_inference_graph.py

```diff
@@ -107,7 +112,10 @@ def main(_):
         num_classes=(dataset.num_classes - FLAGS.labels_offset),
         is_training=FLAGS.is_training)
     if hasattr(network_fn, 'default_image_size'):
-      image_size = network_fn.default_image_size
+      if FLAGS.override_default_image_size:
+        image_size = FLAGS.default_image_size
+      else:
+        image_size = network_fn.default_image_size
```

Comment:
There is the [`eval_image_size` flag](https://github.com/tensorflow/models/blob/master/slim/eval_image_classifier.py#L79-L80) in `eval_image_classifier.py` which defaults to `None`. Perhaps something like [this line](https://github.com/tensorflow/models/blob/master/slim/eval_image_classifier.py#L126) could be a cleaner solution?

```python
eval_image_size = FLAGS.eval_image_size or network_fn.default_image_size
```

So if the flag is not specified, `image_size` defaults to `network_fn.default_image_size`. Otherwise, it uses the value from `FLAGS.default_image_size`.